tests/support/napalm.py

```diff
@@ -364,12 +364,17 @@
             "level": 15,
             "password": "$1$0P70xKPa$4jt5/10cBTckk6I/w/",
             "sshkeys": [
-                "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC4pFn+shPwTb2yELO4L7NtQrKOJXNeCl1je\
-            l9STXVaGnRAnuc2PXl35vnWmcUq6YbUEcgUTRzzXfmelJKuVJTJIlMXii7h2xkbQp0YZIEs4P\
-            8ipwnRBAxFfk/ZcDsdfsdfsdfsdN56ejk345jhk345jk345jk341p3A/9LIL7l6YewLBCwJj6\
-            D+fWSJ0/YW+7oH17Fk2HH+tw0L5PcWLHkwA4t60iXn16qDbIk/ze6jv2hDGdCdz7oYQeCE55C\
-            CHOHMJWYfN3jcL4s0qv8/u6Ka1FVkV7iMmro7ChThoV/5snI4Ljf2wKqgHH7TfNaCfpU0WvHA\
-            nTs8zhOrGScSrtb mircea@master-roshi"
+                "ssh-rsa"
```

Comment:
Not sure if this is correct.  I don't think we want those white space lines in there, the SSH key should be one long continuous line right?

Reply:
Nice find. Will be fixed on next push.